cmd/aigw/run.go

```diff
@@ -100,7 +100,10 @@ func run(ctx context.Context, c cmdRun, stdout, stderr io.Writer) error {
 		return fmt.Errorf("failed to execute certgen: %w: %s", err, certGenOut.String())
 	}
 
-	tmpdir := os.TempDir()
+	tmpdir := filepath.Join(os.TempDir(), "aigw-run")
```

Comment:
this is also important and tangentially relevant; top level tmpdir can be troublesome (maybe only on macos)